deb/env.txt

```diff
@@ -0,0 +1,6 @@
+JJS_SYSROOT=/var/lib/jjs
+JJS_PATH=/usr
+DATABASE_URL=postgres://jjs:internal@localhost:5432/jjs
```

Comment:
Add TODO: we should default to unix socket instead

Reply:
Is it valid to have comments in systemd env. files?

Reply:
Well, add to any other place then:)

Reply:
Anyway I don't understand why unix socket is better than loopback TCP. Postgres still does have proper authorization.